challenges/objects-arrays.js

```diff
@@ -5,121 +5,276 @@
   Use this pattern to create your objects: 
   object name, diet, weight, length, period
```

Comment:
we are not creating the class dinosaur here, it is just an object, let tyrannosaurs = {}